src/components/common/ColonyFundingBanner/ColonyFundingBanner.tsx

```diff
@@ -0,0 +1,50 @@
+import React, { useMemo } from 'react';
+import { defineMessages, FormattedMessage } from 'react-intl';
+
+import MaskedAddress from '~shared/MaskedAddress';
+import { splitAddress } from '~utils/strings';
+import { useColonyContext } from '~hooks';
+
+import styles from './ColonyFundingBanner.css';
+
+const displayName = 'common.ColonyFundingBanner';
+
+const MSG = defineMessages({
+  tipText: {
+    id: `${displayName}.tipText`,
+    defaultMessage: `Tip: to fund your colony, send tokens to your colony's address:`,
+  },
+});
+
+const ColonyFundingBanner = () => {
+  const { colony } = useColonyContext();
+  const { colonyAddress = '' } = colony || {};
+
+  const formattedAddress = useMemo(() => {
```

Comment:
1. See https://github.com/JoinColony/colonyCDapp/pull/74#discussion_r1131145391
2. Not really, since this is the only place in the whole Dapp/Cdapp this is being used